src/storage_eeprom.c

```diff
@@ -74,13 +76,23 @@ int thingset_storage_load()
         size_t processed_size = 0;
         size_t total_read_size = sizeof(header);
         size_t len = header.data_len;
+        size_t read_offset = 0;
         do {
             int size = len > sbuf->size ? sbuf->size : len;
-            LOG_DBG("Reading %d bytes starting at offset %d", size, total_read_size);
-            err = eeprom_read(eeprom_dev, total_read_size, sbuf->data, size);
-            if (err) {
-                LOG_ERR("Error %d reading EEPROM.", -err);
-                break;
+
+            read_offset = total_read_size;
+            int index = DIV_ROUND_UP(size, MAX_READ_SIZE);
```

Comment:
Can we choose a more clear variable name? What about this:
```suggestion
            int num_chunks = DIV_ROUND_UP(size, MAX_READ_SIZE);
```